InterfaceParser/requirements.txt

```diff
@@ -0,0 +1,3 @@
+xmlschema
+pylint
```

Comment:
Thank you for noting the required dependencies, but could you also mention these packages in the README?

Reply:
@JackLivio these requirements is not for XML Parser code, those requirements for `InterfaceParser/runner.py` example and unit tests. Should we still include this in the README?

Reply:
@vladmu Yes Please. But it can be noted that the depdencies are only for example and unit tests.

Reply:
@JackLivio done in 3839526e49238388103d4ee719afc925a3f9c1bf